test/helpers/kubectl.ts

```diff
@@ -102,6 +108,22 @@ export async function getPodLogs(podName: string, namespace: string): Promise<an
   return logsOutput.stdout;
 }
 
+export async function waitForDeployment(name: string, namespace: string): Promise<void> {
+  console.log(`Trying to find deployment ${name} in namespace ${namespace}`);
+  for (let attempt = 0; attempt < 60; attempt++) {
+    try {
+      await exec(`./kubectl get deployment.apps/${name} -n ${namespace}`);
+    } catch (error) {
+      await sleep(1000);
+    }
+  }
+  console.log(`Found deployment ${name} in namespace ${namespace}`);
```

Comment:
should we throw something in case we actually couldn't find the deployment? currently we print that anyway

Reply:
It would be accurate indeed.. right now it would fail later, when it tries to wait for a deployment named `undefined` - it will fail immediately